# NOTE: You have to change this node name if you change the node name in the launchfile.
segment_from_point:
  ros__parameters:
    # The name of the Segment Anything model checkpoint to use
    model_name: sam_vit_b_01ec64.pth
    # The URL to download the model checkpoint from if it is not already downloaded
    model_base_url: "https://dl.fbaipublicfiles.com/segment_anything/"
    # The number of contender masks to return
    n_contender_masks: 3
    # The rate at which to send feedback to the action client
    send_feedback_hz: 10 # Hz